lib/cli/interactive-setup/aws-credentials.js

```diff
@@ -352,9 +352,13 @@ module.exports = {
       context.inapplicabilityReasonCode = 'NON_AWS_PROVIDER';
       return false;
     }
-    if (new AWS.S3().config.credentials) {
-      context.inapplicabilityReasonCode = 'LOCAL_CREDENTIALS_CONFIGURED';
-      return false;
+    try {
+      if (await new AWS.S3({}).config.credentials()) {
```

Comment:
I know it's a slightly bigger change, but this looks to be doing that `aws/has-local-credentials` is doing so I think it could make sense to just use that instead - then you could remove the `try/catch`.